examples/2d/contributors.rs

```diff
@@ -264,7 +259,7 @@ fn collision_system(
     let wall_left = -(window.width() / 2.);
     let wall_right = window.width() / 2.;
 
-    for (mut velocity, mut transform) in query.iter_mut() {
+    query.for_each_mut(|(mut velocity, mut transform)| {
```

Comment:
Because this isn't a proper benchmark, I would probably avoid making this change to prioritize readability.

Reply:
Ditto below.

Reply:
Sounds good, done!